src/status_im/ui/components/colors.cljs

```diff
@@ -7,9 +7,8 @@
 (def black "#000000") ;; Used as the default text color
 (def black-transparent "#00000020") ;; Used as background color for rounded button on dark background
 (def gray "#939ba1") ;; Used as a background for a light foreground and as section header and secondary text color
-(def gray-light "#d9dae1") ;; Used as divider color
+(def gray-light "#E8EBEC") ;; Used as divider color
```

Comment:
I know it's nitpicking, but can you change alpha digits to lowercase to keep it consistent?

Reply:
This will be rebased as @alwx made the change in another PR too.

Reply:
Done